config/locales/en.yml

```diff
@@ -256,6 +256,7 @@ en:
       chilly_bin: "Chilly Bin"
       search_results: "Search Results"
       epic: 'Epic'
+      history: 'History Of'
```

Comment:
the key is `history` and the value is `history of`. What do you think of renaming it to `history_of`?